Products/CMFPlone/browser/login/templates/mail_password_template.pt

```diff
@@ -33,7 +33,7 @@ The following link will take you to a page where you can reset your password for
      i18n:translate="mailtemplate_tracking_information"
      tal:condition="isAnon">
 If you didn't expect to receive this email, please ignore it. Your password has not been changed.
-Request made from IP address <tal:i18n tal:define="host request/HTTP_X_FORWARDED_FOR|request/REMOTE_ADDR"
```

Comment:
This tal expression means get `request/REMOTE_ADDR` if request/HTTP_X_FORWARDED_FOR raises an exception, which is never the case!